apps/judicial-system/backend/src/app/modules/file/file.controller.ts

```diff
@@ -98,7 +107,7 @@ export class FileController {
     return this.fileService.createPresignedPost(theCase, createPresignedPost)
   }
 
-  @UseGuards(RolesGuard, CaseExistsGuard, CaseWriteGuard)
+  @UseGuards(RolesGuard, CaseExistsGuard, CaseWriteGuard, WriteCaseFileGuard)
```

Comment:
Generally, we have not restricted access to file uploads for roles that are part of the police, prosecutionand courts. I we start doing that, then we should also restrict access to file deleting and updating. For now, I would not like to go down that road.